sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubOptions.cs

```diff
@@ -384,7 +384,7 @@ public string Format()
             new JObject
         {
             { nameof(EventHubConnectionOptions.TransportType), ConnectionOptions.TransportType.ToString() },
-            { nameof(EventHubConnectionOptions.Proxy), ConnectionOptions.Proxy?.ToString()},
+            { nameof(EventHubConnectionOptions.Proxy), ConnectionOptions.Proxy?.ToString() ?? string.Empty},
```

Comment:
Fixing so that it won't display "Proxy: null"

Reply:
Why is null bad?

Reply:
It just seemed wrong when we are representing every setting as a string

Reply:
Was it `null` or `"null"`?

Reply:
it was "Proxy: null"

Reply:
Might be better than empty, actually.